### ISSUE TITLE
NullBooleanField is deprecated in Django 3.1 for Checkbox field type

### ISSUE THREAD

**Issue by monnix** [Thermondo/django-heroku-connect#99]
I'm getting the following warning from Heroku:
```
Account.is_deleted: (fields.W903) NullBooleanField is deprecated. Support for it (except in historical migrations) will
be removed in Django 4.0. HINT: Use BooleanField(null=True) instead.
```

When I looked at the declaration for Checkbox in the source I see:
```
class Checkbox(HerokuConnectFieldMixin, models.NullBooleanField):
"""Salesforce ``Checkbox`` field."""
pass
```

Can you folks update the code or do I need to fork it?

**Comment by codingjoe** [comment]
Hi @monnix,

Thanks for pointing out the deprecation issue. I am sure @syphar will merge your pull-request, if you want to contribute a patch yourself. Otherwise, I am sure someone will get to this, before Django 4.0 is released.

Best,
Joe

**Comment by amureki** [comment]
Fixed in https://github.com/Thermondo/django-heroku-connect/pull/100 and released as [0.22.0](https://pypi.org/project/django-heroku-connect/0.22.0/).
Thank you for your help @monnix and @codingjoe ! ❤️
